.github/workflows/pull-request.yml

```diff
@@ -27,81 +29,81 @@ env:
 
 jobs:
 
-  supported-reqs:
+  build-basic:
 
-    name: 'Supported scripts/requirements.txt'
+    name: "build with requirements.txt"
     runs-on: ubuntu-20.04
 
     steps:
       - uses: actions/checkout@v2
 
-      - name: apt-get install
-        run: sudo apt-get -y install $ubuntu_base_deps
-
-      - name: PATH += .local/bin
-        run: echo "$HOME/.local/bin" >> $GITHUB_PATH
+      - name: setup env with requirements.txt
+        run: |
+          sudo apt-get -y install $ubuntu_base_deps
+          echo "$HOME/.local/bin" >> $GITHUB_PATH
+          pip install -r scripts/requirements.txt
 
-      - name: 'pip install -r scripts/requirements.txt'
-        run: pip install -r scripts/requirements.txt
+      - name: build doc
+        run: ./.github/actions/build-sof-doc.sh
 
-      # No YAML anchors; this is copy/paste
-      - name: configure SOF doc
+      - name: prepare file for deploy
+        if: ${{ github.event_name == 'push' && github.ref == 'refs/heads/publish' }}
         run: |
-          git clone https://github.com/thesofproject/sof
-          cmake -GNinja -S sof/doc -B _build_doxy
+          mkdir _build/latest
+          mv -v _build/html/* _build/latest
```

Comment:
I think you're missing `_build_doxy`, the doxygen APIs are probably missing.

The other option is to change `_build_doxy` to `_build/doxy/`. Avoid `_build/doxygen/` because then there will be a `_build/doxygen/doxygen`

Reply:
Just checked and the APIs are NOT missing at https://xiulipan.github.io/latest/api/uuid-api.html. Strange.

Reply:
The sod-docs build step used the doxygen output in its build process. We do not need them in final deploy stage.